Examples/UIExplorer/js/WebViewExample.js

```diff
@@ -183,6 +184,78 @@ class Button extends React.Component {
   }
 }
 
+class ControlledWebView extends React.Component {
```

Comment:
Controlled might not be the best name as this also means something else: https://facebook.github.io/react/docs/forms.html#controlled-components